xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-test/xwiki-platform-flamingo-skin-test-pageobjects/src/main/java/org/xwiki/flamingo/sking/test/po/ExportModal.java

```diff
@@ -49,8 +51,15 @@ public ExportModal()
      */
     public OtherFormatPane openOtherFormatPane()
     {
-        getDriver().findElementByLinkText(OTHER_FORMAT_PANE_LINK).click();
-        getDriver().waitUntilElementIsVisible(By.cssSelector("#exportModelOtherCollapse .panel-body"));
+        WebElement linkPane = getDriver().findElementByLinkText(OTHER_FORMAT_PANE_LINK);
+        linkPane.click();
+
+        // sometimes it looks like the first click does not work.
+        if (linkPane.getAttribute("class").contains("collapsed")) {
+            linkPane.click();
+        }
+        By paneContent = By.className(CONTAINER_TREE_CLASS);
+        getDriver().waitUntilElementIsVisible(paneContent);
```

Comment:
AFAICS the pane content is displayed right away but there is some animation and some opacity transition. If you check the class it changes from:

    collapse -> collapsing -> collapse in

So you should probably wait until the pane content has class "collapse in".

Reply:
I can change that, but anyway the bug is not related to that: I got a 20 sec timeout and when looking on the screenshot of the failing test, you can see that the pane is not displayed at all, as if the click hasn't been done.